src/Primitives/src/StringSegment.cs

```diff
@@ -220,9 +220,9 @@ public bool Equals(string text)
         /// </summary>
         /// <param name="text">The <see cref="string"/> to compare with the current <see cref="StringSegment"/>.</param>
         /// <param name="comparisonType">One of the enumeration values that specifies the rules to use in the comparison.</param>
-        /// <returns><code>true</code> if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>; otherwise, <code>false</code>.</returns>
+        /// <returns><c>true</c> if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>; otherwise, <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if the specified <see cref="string"/> is equal to the current <see cref="StringSegment"/>; otherwise, <see langword="false" />.</returns>
```